client/src/app/ui/modules/list/components/sort-filter-bar/sort-filter-bar.component.ts

```diff
@@ -174,6 +177,14 @@ export class SortFilterBarComponent<V extends Identifiable> {
         private bottomSheet: MatBottomSheet
     ) {}
 
+    public ngOnInit() {
+        this.vp.isMobileSubject.subscribe(v => {
+            if (v) {
+                this.searchEdit = false;
+            }
+        });
+    }
+
```

Comment:
The main intention of my last cr was that I think this subscription could be fully deleted with the proposed change. 

If you think that this is not the case you still need to close the subscription when the component is destroyed.

Reply:
Done, I have add an unsubscribe.